schema/dist/types.ts

```diff
@@ -0,0 +1,169 @@
+export type Maybe<T> = T | undefined;
+export type InputMaybe<T> = T | undefined;
```

Comment:
Do we have null checking enabled in the TSConfig? Also, how is this different from the `Maybe<T>`?